cfgov/unprocessed/css/images.less

```diff
@@ -0,0 +1,7 @@
+// Richtext images have set width and height attributes. Reset them so the
+// images fills it's container without distortion.
```

Comment:
First, two pedantic things:
1. I'd open with "Images inserted in rich text fields have …".
2. Should be "image [singular] fills its [no apostrophe] container".

More importantly, it seems somewhat inaccurate. Images less wide than their container will not be scaled up to the width of their container, right?

Finally, it seems very possible that the default behavior could change with the implementation of Draftail in Wagtail 2.0. I suggest leaving a to-do to revisit that upon upgrading to 2.0.